wdio/specs/homepage.js

```diff
@@ -5,6 +5,40 @@ describe('Municipality homepage', () => {
     browser.url('/municipality-1/node/1?language=he');
   })
 
+  const checkSelectedLanguage = (language) => {
+    browser.waitForVisible('.btn-group.languages');
+    const activeLanguage= browser.getText('.btn-group.languages .btn.btn-default.active');
+    const languageText = {
+      'hebrew': 'עברית',
+      'arabic': 'العربية',
+      'english':'English',
+    };
+    assert(languageText[language] == activeLanguage, `Expected page language is ${language}, but instead it's ${activeLanguage}`);
+  };
+
+  const checkSelectedUserType = (userType) => {
+    browser.waitForVisible('.btn-group.user-types');
+    const user = browser.getText('.btn-group.user-types .btn.btn-default.active');
+    const userTypeText = {
```

Comment:
would be cleaner to get two arguments `(userType, language)` instead of this string.